packages/keyring-controller/src/KeyringController.ts

```diff
@@ -758,6 +783,15 @@ export class KeyringController extends BaseController<
     });
   }
 
+  async createKeyringFromMnemonic(mnemonic: string): Promise<string> {
```

Comment:
Mnemonics are usually handled as `UInt8Array` for security reasons. Can we do the same here?